addons/product_expiry/models/product_product.py

```diff
@@ -7,11 +7,19 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    life_time = fields.Integer(string='Product Life Time',
-        help='Number of days before the goods may become dangerous and must not be consumed. It will be computed on the lot/serial number.')
-    use_time = fields.Integer(string='Product Use Time',
-        help='Number of days before the goods starts deteriorating, without being dangerous yet. It will be computed using the lot/serial number.')
-    removal_time = fields.Integer(string='Product Removal Time',
-        help='Number of days before the goods should be removed from the stock. It will be computed on the lot/serial number.')
-    alert_time = fields.Integer(string='Product Alert Time',
-        help='Number of days before an alert should be raised on the lot/serial number.')
+    expiration_date = fields.Boolean(string='Expiration Date',
```

Comment:
would be perfect if
- expiration_date become _use_expiration_date
- life_time -> expiration_time
- life_date -> expiration_date